src/main/kotlin/no/nav/syfo/simba/MapInntektsmelding.kt

```diff
@@ -25,8 +26,8 @@ fun mapInntektsmelding(arkivreferanse: String, aktorId: String, journalpostId: S
         JournalStatus.FERDIGSTILT,
         imd.arbeidsgiverperioder.map { t -> Periode(t.fom, t.tom) },
         imd.beregnetInntekt,
-        Refusjon(imd.refusjon.refusjonPrMnd ?: 0.0.toBigDecimal()),
-        emptyList(),
+        Refusjon(imd.refusjon.refusjonPrMnd ?: 0.0.toBigDecimal(), imd.refusjon.refusjonOpphører),
+        imd.refusjon.refusjonEndringer?.map { EndringIRefusjon(it.dato, it.beløp) } ?: emptyList(),
```

Comment:
Er det ikke nødvendig med .toList() her? er map og list interchangable?
cc @bjerga

Reply:
Her problemmen er i det at refusjonsendringer er nullable SIMBA modelen men ikke nullable i SPINN modelen. Så elvis operatoren er nødvendig for det.

Reply:
Det er greit. Men jeg lurer på når den ikke er tom. Vil da .map returnere en map? Og er det problematisk siden den forventer en liste  List<EndringIRefusjon>

Reply:
Map returnerer List<>?

Reply:
Map returnerer Liste ikke Map eller jeg ikke kanskje forstår  spørsmålet ?

Reply:
`map` er en listefunksjon som mapper elementene i listen til noe annet, så outputen av `map` er en liste.

Forresten så kan man bruke `.orEmpty()` istedenfor `?: emptyList()`.

Reply:
Ok ser nå at jeg har feil her. Det blir til liste.
Da er det greit for meg